### PR TITLE
Add IDEA specific files in .gitignore

### PR DESCRIPTION
**Pull request by vbauer**

### CODE REVIEW

**Comment by vbauer** [comment]
I signed it!

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,7 @@ target
 .settings
 .clover
 .*
+.idea
```
Comment (cpovirk) [inline .gitignore]: Does the `.*` above cover this one?

Comment (cpovirk) [inline .gitignore]: I'm going to merge without `.idea`. If it's necessary, let me know.

Sorry for the huge delay.

Comment (vbauer) [inline .gitignore]: Maybe it will be better to remove `.*` and keep `.idea`?

Comment (cpovirk) [inline .gitignore]: Hmm, could be. I guess we can reconsider when we next add another `.*` file to the repo.

**Pull request closed without merging**